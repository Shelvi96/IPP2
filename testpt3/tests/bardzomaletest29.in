NEW ID

8;:;861:0;>24730;78
0086;22>:
9618912:>2
1858>9
;:0;:0:93>151
42;18140>517;1;01
:62860:14>752;0148
885211520>4382548962
69>80::4
8562;7342>1::107
@8429:38;36105
@5;;4;20125;0;;093816:49:3548
@1561524;6513862;
@;2:1085260;388456
@28126:79:5
@97;46442161154150;;
@1;339637407
@8::
@328
@19::53462784;3;89232728
@16580101320868683646:43;::15
@186;520765:95:7557930718
@561:460443::398179:94
@205:4
@3438287
@2
