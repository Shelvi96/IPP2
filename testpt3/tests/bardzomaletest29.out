3
12566552533833056
520
20101
0
1069443
0
0
0
78026632695
2010100000000000
858294745726
201011000
0
0
0
